docs/content/direct/architecture.md

```diff
@@ -61,14 +61,19 @@ and updates `WorkStatus` objects in the ITS namespace associated with the WEC.
 
 ## KubeStellar Controller Manager
 
-This module manages binding controller, and status controller. 
+This module manages the binding controller and the status controller. 
 
-* The binding controller watches `BindingPolicy` and workload objects on the Workload Definition Space (WDS), and maintains
-a `Binding` object for each `BindingPolicy` in the WDS. The `Binding` object contains references to the concrete list of
-workload objects and references to the concrete list of clusters that were selected by the `BindingPolicy` selectors.
+* The binding controller watches `BindingPolicy` and workload objects
+on the Workload Definition Space (WDS), and maintains a `Binding`
+object for each `BindingPolicy` in the WDS. A `Binding` object
+contains (a) the concrete list of references to workload objects (and
+associated modulations on downsync behavior) and (b) the concrete list
```

Comment:
Perhaps should add a hyperlink to the mentioned modulations on downsync behavior.

Reply:
I am not aware of a good place to link to. #2463 should make this easier. Right now, I _could_ list them three inline here. But I think that is getting into more detail than is appropriate here (a section about the controller-manager).